test/webhdfs/fileutils.rb

```diff
@@ -1,67 +1,80 @@
 require 'test_helper'
```

Comment:
It looks like this is the only file still using test-unit? I think it wouldn't take too long to just copy this into an rspec file, and then we'd only be using 1 testing library, which seems like a good idea. Do you think that's reasonable?

Reply:
I think that's reasonable and that was my plan which I originally included in the OP checklist, but I forgot about it. I'll definitely do that!

Reply:
Implemented in c409eb0d82fe85146dbe2ed3b1ada9f2bf06bfa7